web/modules/custom/paragraph_align/paragraph_align.routing.yml

```diff
@@ -0,0 +1,7 @@
+paragraph_align:
+  path: '/paragraph/page'
+  defaults:
+    _controller: '\Drupal\paragraph_align\Controller\ParagraphController::para'
+    _title: 'Welcome to My Module in Drupal 9'
```

Comment:
_title: 'Welcome to My Module in Drupal 9'?
I reported about this issue in a previous MR.